Source/WebCore/platform/network/ResourceRequestBase.h

```diff
@@ -280,6 +282,9 @@ class ResourceRequestBase {
     bool isPrivateTokenUsageByThirdPartyAllowed() const { return m_requestData.m_isPrivateTokenUsageByThirdPartyAllowed; }
     void setIsPrivateTokenUsageByThirdPartyAllowed(bool);
 
+    bool wasSchemeOptimisticallyUpgraded() const { return m_requestData.m_wasSchemeOptimisticallyUpgraded; }
+    void setwasSchemeOptimisticallyUpgraded(bool);
```

Comment:
```suggestion
    void setWasSchemeOptimisticallyUpgraded(bool);
```

Reply:
Good catch, that was a bad replace-all substitution.